app/src/main/java/com/sopt/now/compose/data/AuthRepositoryImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.sopt.now.compose.data
+
+import com.sopt.now.compose.datasource.AuthService
+import com.sopt.now.compose.model.login.RequestLoginDto
+import com.sopt.now.compose.model.signup.RequestSignUpDto
+import com.sopt.now.compose.repository.AuthRepository
+import com.sopt.now.compose.utils.BaseResponse
+import retrofit2.Response
+
+class AuthRepositoryImpl(
+    private val authService: AuthService
+) : AuthRepository {
+    override suspend fun loginUser(data: RequestLoginDto): Response<BaseResponse<Unit>> {
```

Comment:
```suggestion
    override suspend fun loginUser(data: RequestLoginDto): Response<BaseResponse<Unit>> =runCatching {
        authService.login(data)
    }
```
이처럼 하면 runCatch에 해당하는 로직을 여기에 넣을 수 도 있어요!